docs/LinuxTool/tree.md

```diff
@@ -2,8 +2,140 @@
 
 ## 1. 基本功用
 
+tree是linux系统下十分常用的一个小工具。其基本作用是用于以树状图的方式显示指定目录下存在的各级子目录和子文件。只使用linux内建的ls指令虽然能够获取任意目录下子目录和子文件的信息，但是却不能够展示完整地展示多层次的目录、文件及之间的关系。在阅览较为复杂的工程时，tree的功能就显得非常重要了,能够帮助我们对整个工程各部分的文件的关系有直观清晰的认识。
+
```

Comment:
You were meaning that no dot should appear in the whole file at last review. I finally start to understand!

Reply:
For example:

`你好。欢迎你来贡献。` should be changed to `你好。欢迎你来贡献`

Reply:
oh, i may consider that quite weird for me. However, I will follow what you said.

Reply:
We often make a singel sentence to be an individual line so that no dot at the end will make it look simple and beautiful. But this convention actully depends on the project itself. This is a documentation which uses `chinese` as spoken-language, and the end-dot is more apparent(so I recommend not to have it). But when come to an all-English project, you may be recommended to use dot at the end.

Reply:
@nagi2330 I unresolve this to make sure you have read it and the explaination above

And you should reply to say something like `resolve in commit(<sha of the commit>)` use the sha to link to the commit before you resolve conversation(to make sure you haven't resolve by accident)

Reply:
resolve in commit(9369f9e)